simulator/src/main/java/com/hazelcast/simulator/provisioner/Provisioner.java

```diff
@@ -189,14 +193,14 @@ public void scale(int size, boolean enterpriseEnabled) throws Exception {
             echo("Current number of machines: " + addresses.size());
             echo("Desired number of machines: " + (addresses.size() + delta));
             echo("Ignoring spawn machines, desired number of machines already exists.");
-        } else if (delta < 0) {
-            terminate(-delta);
+        } else if (delta > 0) {
+            create(delta, enterpriseEnabled);
         } else {
-            scaleUp(delta, enterpriseEnabled);
+            terminate(-delta);
         }
     }
 
-    private void scaleUp(int delta, boolean enterpriseEnabled) throws Exception {
+    private void create(int delta, boolean enterpriseEnabled) throws Exception {
```

Comment:
Good idea, will do so. We still have `terminate()` method which will call `scaleDown(Integer.MAX_VALUE);` :)